### ISSUE TITLE
Resolve Accessibility issues highlighted by Google Lighthouse Audits

### ISSUE THREAD

**Issue by psyked** [internetarchive/openlibrary#1005]
Lighthouse: https://developers.google.com/web/tools/lighthouse/ highlights a few issues with general page accessibility. Currently it is highlighting issues with:

- [x] Duplicate element Ids in the footer links
- [ ] Insufficient color contrast ratios for some links
- [x] Footer pantheon image missing alt tag
- [ ] Form elements missing labels

**Comment by psyked** [comment]
The warnings about "insufficient color contrast ratio" are referring to the Primary button styles, which currently have a background of #3b98fc and a text color of #ffffff, which a fair distance off a AAA or even AA compliant contrast ratio according to https://webaim.org/resources/contrastchecker/

This is going to be a difficult issue to resolve, because these blue buttons are a fairly dominant part of the brand, but it looks like there's an argument for darkening the background color and increasing the font size to improve accessibility.

**Comment by jdlrobson** [comment]
> Form elements missing labels
@psyked which page did you run this on?

I'm able to replicate the contrast issues quite easily, but not the labels...

**Comment by konpyuta** [comment]
Concerning the color contrast issue:
The _Join Waitlist_ buttons (.borrow_unavailable) also do not have enough contrast (#ffa238 bg, #ffffff text). May I recommend using the banner bg color (#026dab) for the buttons, and #CC4E0C for the .borrow-unavailable buttons? Also is darkening the blue in the Open Library logo a possibility?

Here is an example of what how it would look:
<img width="1033" alt="screen shot 2018-10-22 at 10 35 41 pm" src="https://user-images.githubusercontent.com/6303833/47331746-1b143300-d64b-11e8-930b-126fe6f47131.png">

**Comment by jdlrobson** [comment]
Sounds sensible to me!
@mekarpeles any issues with this?

If we do want to make a change it should be straightforward - replace the colors in
https://github.com/internetarchive/openlibrary/blob/master/static/css/less/colors.less

**Comment by konpyuta** [comment]
The only caveat is that the primary button inside the banner might need another class/additional styling. In the screenshot I just put 70% opacity white bg and black text, but you could also make it an outline button.

**Comment by mekarpeles** [comment]
I'm open to colors with more contrast but I want to be mindful not to just "darken until accessible". The darker blue is significantly different than the Open Library logo background and I'd love for us to go in the direction of fewer colors if we're going to make changes.

I'm also not a big fan of the red/orange waitlist button. A solid primary red  (and something closer to a solid primary blue) might look more cohesive.

Would love additional weigh in from folks approaching this issue from an aesthetics position, in addition to accessibility (both valued)

**Comment by mekarpeles** [comment]
Thanks @psyked and @konpyuta for helping push this forward!

**Comment by seabelis** [comment]
You probably want to avoid changing the logo color itself since that is your branding, but it is possible to create a color palette for the site's components that would aesthetically agreeable. Quick examples:
https://color.adobe.com/cloud/aHR0cHM6Ly9jYy1hcGktYXNzZXRzLmFkb2JlLmlv/library/76fb32fc-5611-47f6-a3d6-301ae7d2cd77/theme/6f33ef4d-ca5a-4106-a848-ec38708c1867/
https://color.adobe.com/cloud/aHR0cHM6Ly9jYy1hcGktYXNzZXRzLmFkb2JlLmlv/library/76fb32fc-5611-47f6-a3d6-301ae7d2cd77/theme/5f9123ad-0d2b-40e1-b595-ac4c15254ad9/
https://color.adobe.com/cloud/aHR0cHM6Ly9jYy1hcGktYXNzZXRzLmFkb2JlLmlv/library/76fb32fc-5611-47f6-a3d6-301ae7d2cd77/theme/8643b4ca-42bb-4046-a98d-5f5b0ef50bb5/
https://color.adobe.com/cloud/aHR0cHM6Ly9jYy1hcGktYXNzZXRzLmFkb2JlLmlv/library/76fb32fc-5611-47f6-a3d6-301ae7d2cd77/theme/2658e252-5e82-4c7f-8242-477f361b48ba/

**Comment by guyjeangilles** [comment]
Assigning jdlrobson (not tagging per request). Also labeling the issue as blocked since that's it's category the Modern Front-End Project

**Comment by jdlrobson** [comment]
Form issue no longer applies. Spun out remaining issue to #2610. Let's continue the color conversation there.
